pkg/security/probe/mount_resolver.go

```diff
@@ -426,28 +375,23 @@ func (mr *MountResolver) resolveMount(mountID, pid uint32) (*model.MountEvent, e
 	return mount, nil
 }
 
-// GetMountPath returns the path of a mount identified by its mount ID. The first path is the container mount path if
+// ResolveMountPaths returns the path of a mount identified by its mount ID. The first path is the container mount path if
```

Comment:
This comment needs to be updated to match the new return values

Reply:
Oups, thanks

Reply:
Fixed